sys/include/net/ipv4/addr.h

```diff
@@ -74,15 +75,30 @@ char *ipv4_addr_to_str(char *result, const ipv4_addr_t *addr, uint8_t result_len
  * @brief   Converts an IPv4 address string representation to a byte-represented
  *          IPv4 address
  *
- * @param[in] result    The resulting byte representation
- * @param[in] addr      An IPv4 address string representation
+ * @param[out] result    The resulting byte representation
+ * @param[in] addr       An IPv4 address string representation
  *
  * @return  @p result, on success
  * @return  NULL, if @p addr was malformed
  * @return  NULL, if @p result or @p addr was NULL
  */
 ipv4_addr_t *ipv4_addr_from_str(ipv4_addr_t *result, const char *addr);
 
+/**
+ * @brief   Converts an IPv4 address from a buffer of characters to a
+ *          byte-represented IPv4 address
+ *
+ * @param[out] result    The resulting byte representation
+ * @param[in] addr       An IPv4 address string representation
+ * @param[in] addr_len   The amount of characters to parse
+ *
+ * @return  @p result, on success
+ * @return  NULL, if @p addr was malformed
+ * @return  NULL, if @p result or @p addr was NULL
+ */
+ipv4_addr_t *ipv4_addr_from_buf(ipv4_addr_t *result, const char *addr,
```

Comment:
You could add the range constraint in the documentation, e.g., using `@note`.

Reply:
Added